docs/docs/guides/remote-access.md

```diff
@@ -0,0 +1,56 @@
+:::
+
+## Option 1: VPN to home network
+
+You may use a VPN service to open an encrypted connection to your Immich instance. OpenVPN and Wireguard are two popular VPN solutions. Here is a guide on setting up VPN access to your server - [Pihole documentation](https://docs.pi-hole.net/guides/vpn/wireguard/overview/)
+
+### Pros:
+
+- Simple to set up and very secure.
+- Single point of potential failure, i.e., the VPN software itself. Even if there is a zero-day vulnerability on Immich, you will not be at risk.
+- Both Wireguard and OpenVPN are independently security-audited, so the risk of serious zero-day exploits are minimal.
+
+### Cons:
+
+- If you don't have a static IP address, you would need to set up a [Dynamic DNS](https://www.cloudflare.com/learning/dns/glossary/dynamic-dns/). [DuckDNS](https://www.duckdns.org/) is a free DDNS provider.
+- VPN software needs to be installed and active on both server-side and client-side.
+- Requires you to open a port on your router to your server.
+
+## Option 2: Tailscale
+If you are unable to open a port on your router for Wireguard or OpenVPN to your server, [Tailscale](https://tailscale.com/) is a good option. Tailscale mediates a peer-to-peer wireguard tunnel between your server and remote device, even if one or both of them are behind a [NAT firewall](https://en.wikipedia.org/wiki/Network_address_translation). 
+
+### Pros
+
+- Minimal configuration needed on server and client sides.
+- You are protected against zero-day vulnerabilities on Immich.
+
+### Cons
+- The Tailscale client usually needs to run as root on your devices and it increases the attack surface slightly compared to a minimal Wireguard server. e.g., an [RCE vulnerability](https://github.com/tailscale/tailscale/security/advisories/GHSA-vqp6-rc3h-83cp) was discovered in the Windows Tailscale client in November 2022.
+- Tailscale is a paid service. However, there is a generous [free tier](https://tailscale.com/pricing/) that permits up to 3 users and up to 100 devices.
+- Tailscale needs to be installed and running on both server-side and client-side.
+
+## Option 3: Self-signed certificate
+Using a [self-signed certificate](https://en.wikipedia.org/wiki/Self-signed_certificate) protects you against man-in-the-middle attacks, but exposes your Immich instance to the web.
+
+### Pros
+- No additional software needs to be installed client-side
+
+### Cons
+- Both the Immich web interface and API may be exposed to the internet. Immich is under very active developement and the existence of severe security vulnerabilities cannot be ruled out.
+
+## Option 4: Reverse Proxy
+A reverse proxy is a service that sits between web servers and clients. A reverse proxy increases security by hiding the server IP address, which makes targeted attacks like [DDoS](https://www.cloudflare.com/learning/ddos/what-is-a-ddos-attack/) harder.
```

Comment:
A reverse proxy does not necessarily hide the server IP/make targeted attacks harder. This is only the case if you're using something like cloudflare, but not when you're running a reverse proxy like nginx on your server. Maybe cloudflare and a local proxy server should be separate sections?

Reply:
Thank you, that clarifies some lingering confusion I had about reverse proxies. I made the changes.